ISISScientificComputing/autoreduce
# Update the queue_processor module readme

### Summary of work
* Updated queue_processor module readme to better describe code responsibilities 

### How to test your work
* Read through  documentation changes for grammar and to ensure they make sense

*No associated issue*
